core/commands/pom.xml

```diff
@@ -43,5 +43,13 @@
       <groupId>org.apache.karaf.shell</groupId>
       <artifactId>org.apache.karaf.shell.core</artifactId>
     </dependency>
+    <dependency>
```

Comment:
On another look, we shouldn't have dao and model dependencies here as this commands bundle is part of opennms-core. 

I think we need to move ShowMinions and ShowLocations into new bundle under features/minion/shell/minions